extensions/wsc/src/main/java/org/mule/extension/ws/internal/WscConnectionProvider.java

```diff
@@ -47,6 +47,13 @@
   private String address;
 
   /**
+   * If should use the MTOM protocol to manage the attachments or not.
+   */
+  @Parameter
+  @Optional(defaultValue = "false")
+  private boolean mtomEnabled;
```

Comment:
nope, this is all client configuration and it's directly related with the service you want to connect to.